pa_matchbatch.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+set -e
+
+# get rev hashes in the form [component]|[to_rev]|[url]
+getComponentRevMap() {
+	for componentName in $(for componentFile in $(grep -lv refs/tags configs/components/*.json); do grep -l puppetlabs ${componentFile}; done); do 
```

Comment:
A comment here clarifying the grep stuff would be nice. I think what's going on here is you're looking for components pinned to tags, and of those, filtering out the ones owned by puppetlabs right?

Reply:
Exactly.